### PR TITLE
fix: portfolio ability to modify amounts in the transfer modal

### PR DESCRIPTION
**Pull request by shane-moore**
This PR attempts to close out issue #631 .

## Situation
- When the user fires a click event when attempting to enter a token transfer value, the click event still bubbles up to the `v-select`, which transfers focus to `input.vs__search` (v-select's built in search input field, which is hidden but still exists in the DOM).  This happens before the `focusInput` method is able to designate focus to `input#bridge-input`.  Since focus always has to perpetually move away from `input.vs__search` upon click, this is why the caret is always placed at the end of `input#bridge-input`.

## Changes:
replace `@click` with `@mousedown` on `input#bridge-input` because the `.stop` modifier will stop the firing of the mouseup event and prevent focus being initially shifted to `input.vs__search` before  `focusInput` moves focus to `input#bridge-input`.

### CODE REVIEW

**Comment by shane-moore** [comment]
tagging @bangjelkoski @ThomasRalee for awareness

**Comment by bangjelkoski** [comment]
Awesome solution. Have you tested this on mobile?

**Comment by shane-moore** [comment]
@bangjelkoski, mobile is a strong consideration, as people should be able to trade anywhere & anytime.
In short: yes, it works.
TLDR: I created an ngrok tunnel to localhost serving the dex, plugged the ngrok provided url into metamask ios browser, and was successful in adjusting the caret location in the transfer modal input field

**Comment by bangjelkoski** [comment]
> @bangjelkoski, mobile is a strong consideration, as people should be able to trade anywhere & anytime.
> 
> In short: yes, it works.
> 
> TLDR: I created an ngrok tunnel to localhost serving the dex, plugged the ngrok provided url into metamask ios browser, and was successful in adjusting the caret location in the transfer modal input field

Nice one!
